tasks/prepare_scenario.yml

```diff
@@ -1,26 +1,36 @@
 ---
 
+# We should store scenario in separated variable to
+# avoid conflicts between plays.
+- name: Set scenario
+  set_fact:
+    scenario: '{{ cartridge_scenario }}'
```

Comment:
You should to add information about new output facts in [scenario.md](/tarantool/ansible-cartridge/blob/fix-prepare-scenario/doc/scenario.md)